src/Microsoft.VisualStudio.ProjectSystem.CSharp.VS/ProjectSystem/VS/Properties/CSharpProjectDesignerPageProvider.cs

```diff
@@ -36,7 +36,10 @@ public Task<IReadOnlyCollection<IPageMetadata>> GetPagesAsync()
             }
 
             builder.Add(CSharpProjectDesignerPage.Debug);
-            builder.Add(CSharpProjectDesignerPage.ReferencePaths);
+
+            // Removed until the page is revamped, see: https://github.com/dotnet/roslyn-project-system/issues/1397.
```

Comment:
I'd say remove this comment. The issue is tracking the work and the commented out code is not adding any value.